examples/react/start-workos/app/routes/logout.tsx

```diff
@@ -0,0 +1,12 @@
+import { createFileRoute } from '@tanstack/react-router';
+import { createServerFn } from '@tanstack/react-start';
+import { signOut } from '../authkit/serverFunctions';
+
+const logoutFn = createServerFn().handler(async () => {
```

Comment:
Any reason why a server function is being wrapped in another server function? Or is this just to get the `GET` method behaviour?

Reply:
I think I just didn't realize that the `loader` could just call the function directly. I simplified it.